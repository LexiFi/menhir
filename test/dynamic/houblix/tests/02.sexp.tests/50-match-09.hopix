let test =
    switch (K ("OK")) {
    | K (x & y) ->
      print_string ("This test is ");
      print_string (x);
      print_string (" and ");
      print_string (y);
      print_string (".\n")
    }