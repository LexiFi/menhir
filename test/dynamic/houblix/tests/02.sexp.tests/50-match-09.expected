(((value(DefineValue(SimpleValue(((value(Id test))(position((start_p(""1 0 4))(end_p(""1 0 8)))))()((value(Case((value(Tagged((value(KId K))(position((start_p(""2 11 23))(end_p(""2 11 24)))))()(((value(Literal((value(LString OK))(position((start_p(""2 11 29))(end_p(""2 11 30)))))))(position((start_p(""2 11 29))(end_p(""2 11 30))))))))(position((start_p(""2 11 23))(end_p(""2 11 31)))))(((value(Branch((value(PTaggedValue((value(KId K))(position((start_p(""3 35 41))(end_p(""3 35 42)))))()(((value(PAnd(((value(PVariable((value(Id x))(position((start_p(""3 35 44))(end_p(""3 35 45)))))))(position((start_p(""3 35 44))(end_p(""3 35 45)))))((value(PVariable((value(Id y))(position((start_p(""3 35 48))(end_p(""3 35 49)))))))(position((start_p(""3 35 48))(end_p(""3 35 49))))))))(position((start_p(""3 35 44))(end_p(""3 35 49))))))))(position((start_p(""3 35 41))(end_p(""3 35 50)))))((value(Sequence(((value(Apply((value(Variable((value(Id print_string))(position((start_p(""4 54 60))(end_p(""4 54 72)))))()))(position((start_p(""4 54 60))(end_p(""4 54 72)))))((value(Literal((value(LString"This test is "))(position((start_p(""4 54 88))(end_p(""4 54 89)))))))(position((start_p(""4 54 73))(end_p(""4 54 90)))))))(position((start_p(""4 54 60))(end_p(""4 54 90)))))((value(Sequence(((value(Apply((value(Variable((value(Id print_string))(position((start_p(""5 92 98))(end_p(""5 92 110)))))()))(position((start_p(""5 92 98))(end_p(""5 92 110)))))((value(Variable((value(Id x))(position((start_p(""5 92 112))(end_p(""5 92 113)))))()))(position((start_p(""5 92 111))(end_p(""5 92 114)))))))(position((start_p(""5 92 98))(end_p(""5 92 114)))))((value(Sequence(((value(Apply((value(Variable((value(Id print_string))(position((start_p(""6 116 122))(end_p(""6 116 134)))))()))(position((start_p(""6 116 122))(end_p(""6 116 134)))))((value(Literal((value(LString" and "))(position((start_p(""6 116 142))(end_p(""6 116 143)))))))(position((start_p(""6 116 135))(end_p(""6 116 144)))))))(position((start_p(""6 116 122))(end_p(""6 116 144)))))((value(Sequence(((value(Apply((value(Variable((value(Id print_string))(position((start_p(""7 146 152))(end_p(""7 146 164)))))()))(position((start_p(""7 146 152))(end_p(""7 146 164)))))((value(Variable((value(Id y))(position((start_p(""7 146 166))(end_p(""7 146 167)))))()))(position((start_p(""7 146 165))(end_p(""7 146 168)))))))(position((start_p(""7 146 152))(end_p(""7 146 168)))))((value(Apply((value(Variable((value(Id print_string))(position((start_p(""8 170 176))(end_p(""8 170 188)))))()))(position((start_p(""8 170 176))(end_p(""8 170 188)))))((value(Literal((value(LString".\n"))(position((start_p(""8 170 194))(end_p(""8 170 195)))))))(position((start_p(""8 170 189))(end_p(""8 170 196)))))))(position((start_p(""8 170 176))(end_p(""8 170 196))))))))(position((start_p(""7 146 152))(end_p(""8 170 196))))))))(position((start_p(""6 116 122))(end_p(""8 170 196))))))))(position((start_p(""5 92 98))(end_p(""8 170 196))))))))(position((start_p(""4 54 60))(end_p(""8 170 196)))))))(position((start_p(""3 35 41))(end_p(""8 170 196))))))))(position((start_p(""2 11 15))(end_p(""9 197 202)))))))))(position((start_p(""1 0 0))(end_p(""9 197 202))))))
